### PR TITLE
chore: fix LaunchOptions proxy parsing

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright-dotnet/issues/2494

### CODE REVIEW

File: src/Playwright.TestAdapter/PlaywrightSettingsXml.cs [hunk 1]
```diff
@@ -82,16 +82,30 @@ private static T ParseXmlIntoClass<T>(XmlReader reader) where T : class, new()
             if (reader.NodeType == XmlNodeType.Element)
             {
                 var key = reader.Name;
-                reader.Read();
-                if (reader.NodeType != XmlNodeType.Text)
+                var property = classType.GetProperty(key);
+                if (property == null)
                 {
-                    Console.Error.WriteLine($"Playwright RunSettings Parsing Error: Playwright>{endTag}>{key} is not supported");
+                    Console.WriteLine($"Playwright RunSettings Parsing Error: Playwright>{endTag}>{key} is not supported");
```
Comment (mxschmitt) [inline src/Playwright.TestAdapter/PlaywrightSettingsXml.cs:88]: note: messages to the stderr don't get shown, hence changing it to stdout as a drive-by.

Comment (yury-s) [reply src/Playwright.TestAdapter/PlaywrightSettingsXml.cs:88]: This sounds wrong, why is it not shown?

File: src/Playwright.TestAdapter/PlaywrightSettingsXml.cs [hunk 2]
```diff
@@ -63,16 +63,16 @@ public PlaywrightSettingsXml(XmlReader reader)
...
             }
         }
     }
 
-    private static T ParseXmlIntoClass<T>(XmlReader reader) where T : class, new()
+    private static T ParseXmlIntoClass<T>(Type classType, XmlReader reader)
     {
         var endTag = reader.Name;
-        var options = new T();
+        var options = Activator.CreateInstance(classType);
```
Comment (yury-s) [inline src/Playwright.TestAdapter/PlaywrightSettingsXml.cs:75]: what type was it instantiating before?

**Review by yury-s — APPROVED** [review]
